test/unit/resources/os_test.rb

```diff
@@ -6,6 +6,24 @@
 require 'inspec/resource'
 
 describe 'Inspec::Resources::Os' do
+  it 'verify NamedCleaned with uppercase' do
```

Comment:
Forgive the pun, but can we `Clean` this up a bit?

```ruby
let(:cleaner) { Inspec::Resources::OSResource::NameCleaned }

it 'does something' do
  cleaner.new('blah')
  # some test here
end
```

It feels repetitious to be defining `cleaner` in each test like that. If you don't want to use a `let`, we should at least consolidate two lines per test:

```ruby
it 'does something' do
  name = Inspec::Resources::OSResource::NameCleaned.new('something')
  # the test
end
```